docs/source/Doxyfile

```diff
@@ -58,7 +58,7 @@ PROJECT_LOGO           =
 # entered, it will be relative to the location where doxygen was started. If
 # left blank the current directory will be used.
 
-OUTPUT_DIRECTORY       =
+OUTPUT_DIRECTORY       = _doxygen
```

Comment:
isolating _doxygen output to be able to add to `exclude_patterns` in `conf.py`.  Doesn't save much time, but it's better to do than not.  It basically just tells sphinx "don't look here for files to convert to html".

Reply:
I've been talking with RTD. I've encountered timeout issues in the past but they gave me a time extension to about 20 minutes. The build failed at 24 second mark so timeouts aren't the issue.

I'm going to go ahead and try merging your changes to see what's new anyway.